microservices/object_detection/google_coral/detect.py

```diff
@@ -126,6 +126,7 @@ def set_input(interpreter, size, resize):
   w, h = size
```

Comment:
This mixture of `h`/`height`and `w`/`width` as variables name here can be a bit confusing. I'd try something more explicit to improve readability. Eg. `interpret_h` / `original_h`

Reply:
solved it